agent/util-scripts/pbench-postprocess-tools

```diff
@@ -9,26 +9,24 @@ action=`echo ${script_name#pbench-} | awk -F- '{print $1}'`
 # source the base script
 . "$pbench_bin"/base
 
-if [ "$action" != "start" -a "$action" != "stop" -a "$action" != "postprocess" -a "$action" != "kill" ]; then 
+if [ "$action" != "start" -a "$action" != "stop" -a "$action" != "postprocess" -a "$action" != "kill" ]; then
```

Comment:
Use [[...]] and braces around variable names?

Reply:
Fixed.